### ISSUE TITLE
exports corrupt fbx in ascii format

### ISSUE THREAD

**Issue by XDM-Inc** [FXTD-ODYSSEY/renderdoc2fbx#9]
after playing around with renderdoc i got a popup to export but no app can understand the fbx, 1 because its in ascii and not binary format and the other is because the ascii is corrupt somewhere and no app wants to convert it

**Comment by mikeseese** [comment]
Apparently you can use the Autodesk FBX converter to get a valid FBX that Blender will read (probably other tools too): https://www.autodesk.com/developer-network/platform-technologies/fbx-converter-archives

**Comment by XDM-Inc** [comment]
> Apparently you can use the Autodesk FBX converter to get a valid FBX that Blender will read (probably other tools too): https://www.autodesk.com/developer-network/platform-technologies/fbx-converter-archives

Nope, that tool can't understand it either. All it does is pump out a corrupted 1kb file

**Comment by mikeseese** [comment]
It worked for my meshes 🤷It is an old tool, and seems like an old format so I'm sure mileage may vary.
